### PR TITLE
[windows] fix crypto

### PR DESCRIPTION
**Pull request by cirospaciari**
### What does this PR do?

Fix `crypto.key-objects.test.ts`

- [ ] Documentation or TypeScript types (it's okay to leave the rest blank in this case)
- [x] Code changes

### How did you verify your code works?

Existing tests

### CODE REVIEW

File: src/bun.js/bindings/KeyObject.cpp [hunk 1]
```diff
@@ -381,6 +381,7 @@ JSC::EncodedJSValue KeyObject__createPrivateKey(JSC::JSGlobalObject* globalObjec
                         auto value = pass.value();
                         passphrase.passphrase = const_cast<char*>(value.data());
                         passphrase.passphrase_len = value.length();
+                        pass_holder = std::move(value);
```
Comment (Jarred-Sumner) [inline src/bun.js/bindings/KeyObject.cpp]: This is a use-after-move. 

Maybe PrivateKeyPassphrase should have a wrapper class which owns the WTF::CString?

**Review by Jarred-Sumner — CHANGES_REQUESTED** [review]
A comment

File: src/bun.js/bindings/KeyObject.cpp (hunk 1 above)
Comment (cirospaciari) [reply src/bun.js/bindings/KeyObject.cpp]: fixed here: https://github.com/oven-sh/bun/pull/8797/commits/5be7384f3a4651e11858c95893c4529591e3e0c4

File: src/bun.js/bindings/KeyObject.cpp [hunk 2]
```diff
@@ -128,15 +128,49 @@ struct AsymmetricKeyValueWithDER {
...
+        this->passphrase = nullptr;
+        this->passphrase_len = 0;
+        if (passphraseJSValue.isUndefinedOrNull() || passphraseJSValue.isEmpty()) {
+            this->hasPassphrase = false;
+            return;
+        }
+        this->hasPassphrase = true;
+        if (passphraseJSValue.isString()) {
+            auto passphrase_wtfstr = passphraseJSValue.toWTFString(globalObject);
+            RETURN_IF_EXCEPTION(scope, );
```
Comment (Jarred-Sumner) [inline src/bun.js/bindings/KeyObject.cpp:150]:
```suggestion
            RETURN_IF_EXCEPTION(scope, void());
```

**Review by Jarred-Sumner — APPROVED** [review]

File: src/bun.js/bindings/KeyObject.cpp (hunk 2 above)
Comment (cirospaciari) [reply src/bun.js/bindings/KeyObject.cpp:150]: Will not compile with void()

**Pull request merged**